gcs/gcs.go

```diff
@@ -430,3 +430,8 @@ func (f *Filter) readFullUint64(b *bstream.BStream) (uint64, error) {
 	v := (quotient << f.p) + remainder
 	return v, nil
 }
+
+// Size returns the Filter's size in bytes.
+func (f *Filter) Size() uint64 {
+	return uint64(len(f.filterData))
```

Comment:
This is wrong as it should include the size serialized. Why can't `NBytes` be used for this?

Reply:
My understanding was that filterData is what's actually consuming the memory since it's the one the struct holds.

So you suggest doing `len(f.NBytes())` ?